agent/engine/dependencygraph/graph.go

```diff
@@ -287,10 +284,20 @@ func containerOrderingDependenciesCanResolve(target *apicontainer.Container,
 			// The 'target' container desires to be moved to its 'steady' state.
 			// Allow this only if the dependency container also desires to be in 'steady state'
 			return dependsOnContainerDesiredStatus == dependsOnContainer.GetSteadyStateStatus()
-
 		}
 		return false
 
+	case SuccessCondition:
+		isDependOnContainerStoppedSuccessfully := dependsOnContainer.GetKnownStatus() == apicontainerstatus.ContainerStopped &&
+			*dependsOnContainer.GetKnownExitCode() == 0
```

Comment:
is `0` always and the only return success code?

Reply:
I think, 0 is the standard exit code for success. If the customer mentions some other exit code, then they will have to use the condition 'COMPLETE'.

Reply:
[Yep!](https://www.tldp.org/LDP/abs/html/exit-status.html)